### PR TITLE
hoomd-v3.0.0-beta.1

### PR DESCRIPTION
**Pull request by bdice**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR provides "dev" builds of HOOMD v3.0.0 beta 1.

### CODE REVIEW

**Comment by bdice** [comment]
@conda-forge-admin, please rerender

**Comment by bdice** [comment]
@conda-forge-admin, please rerender

File: recipe/build_hoomd.sh [hunk 1]
```diff
@@ -27,7 +28,7 @@ cmake ../ \
       -DCMAKE_INSTALL_PREFIX=${SP_DIR} \
       -DPYTHON_EXECUTABLE=${PYTHON} \
       -DENABLE_MPI=off \
-      -DENABLE_CUDA=${CUDA_SUPPORT} ${CUDA_CMAKE_OPTIONS} \
+      -DENABLE_GPU=${CUDA_SUPPORT} ${CUDA_CMAKE_OPTIONS} \
```
Comment (jakirkham) [inline recipe/build_hoomd.sh:31]: Should this be quoted?

```suggestion
      -DENABLE_GPU="${CUDA_SUPPORT} ${CUDA_CMAKE_OPTIONS}" \
```

**Comment by jakirkham** [comment]
Might be worth `cat`ing `$SRC_DIR/build-conda/CMakeFiles/CMakeOutput.log` on `cmake` configuration failure. Tacking something like ` || { cat CMakeFiles/CMakeOutput.log; exit 1; }` onto the `cmake` command should do the trick.

**Review by joaander — APPROVED** [review]
Thanks!

**Pull request merged**